docs/sdks/remove-api-key.md

```diff
@@ -31,11 +31,29 @@ from c8 import C8Client
 client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                         email='nemo@nautilus.com', password='xxxxx',
                         geofabric='_system')
+                        
+# Get a list of all the API key ID's names
+apiKeyIds = []
+for api in client.list_all_api_keys():
+    apiKeyIds.append(api["keyid"])
 
-# Remove api key
-remove = client.remove_api_key('id1')
+# Check whether the chosen_api_key exists or not, comparing it to the API key list
```

Comment:
Why is chosen_api_key lowercase here and uppercase in line 42?

Reply:
Hi Diana. PEP8 standard (and JS standard is similar) states that constants should be in upper-case.

We recently changed our code to be as PEP8 compliant as possible.
And certainly, that comment line was bypassed while making the conversion.
Will fix it shortly.